packages/rest/src/rest.server.ts

```diff
@@ -218,6 +219,12 @@ export class RestServer extends Context implements Server, HttpServerLike {
     this._applyExpressSettings();
     this._requestHandler = this._expressApp;
 
+    const expressCtx = new ExpressContext(this);
+    expressCtx.setMiddlewareRegistryOptions({
```

Comment:
I am not comfortable forcing Express context on all LoopBack4 users.

Can you find a way how to make this extension an optional addition please?